### PR TITLE
Estute/add jars to classpath

### PR DESCRIPTION
**Pull request by estute**
CC @edx/devops

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,12 +21,47 @@ Setup
...
+Jenkins will run Groovy code placed in $JENKINS_HOME/init.groovy.d on each boot.
+
+
+Tips/Tricks
+~~~~~~~~~~~
+
+Unlike the Jenkins Script Console, Jenkins-related libraries are not auto-imported,
+so make sure you import the following into your scripts:
+
+.. code:: groovy
```
Comment (michaelyoungstrom) [inline README.rst]: @estute For some reason, I'm not seeing the groovy code when I open the README. Same goes for line 59

File: README.rst [hunk 2]
```diff
@@ -21,12 +21,47 @@ Setup
 ~~~~~
 
 Install python dependencies for acceptance testing (on a Jenkins instance
-running within a Docker container).
+running within a Docker container), as well as various Groovy helper utilities.
 
 .. code:: bash
 
 make requirements
```
Comment (michaelyoungstrom) [inline README.rst]: I know this was already merged, but this should be indented so it appears as code

File: Dockerfile [hunk 3]
```diff
@@ -18,8 +18,11 @@ ARG gid=1000
 RUN groupadd -g ${gid} ${group} \
     && useradd -d "$JENKINS_HOME" -u ${uid} -g ${gid} -m -s /bin/bash ${user}
 
-RUN mkdir -p $JENKINS_HOME/init.groovy.d
+RUN mkdir -p $JENKINS_HOME/init.groovy.d && \
```
Comment (jzoldak) [inline Dockerfile]: nit: for readability - the convention that I've seen on official Dockerfiles up on dockerhub is to put the "\\\" on the first line, and the "&&" on the continuing line. 
Couple references:
https://hub.docker.com/r/selenium/node-firefox/~/dockerfile/
https://github.com/docker-library/python/blob/1ca4a57b20a2f66328e5ef72df866f701c0cd306/2.7/Dockerfile

**Comment by estute** [comment]
Including the plugin jpi plugin (for plugin dependency management) was causing the assemble task (which is part of codenarc) to be overridden and trying to apply stricter java/groovy compilation rules. I have separated the plugin gradle code into a separate 'plugins.gradle' file and it is called explicitly in the makefile.

**Comment by michaelyoungstrom** [comment]
LGTM @estute

**Pull request merged**